readme.md

```diff
@@ -62,16 +67,16 @@ body{
 All the steps for one way of doing the above have been written below, but in the wrong order. Put them in the correct order.
 
 ```
-$ touch README.md
-$ git push origin master
-$ git push origin master
-$ git remote add origin git@github.com/username/project-repo.git
-$ git remote add juan git@github.com/juan/project-repo.git
-$ git commit -m "initial commit"
 $ cd project-repo
+$ touch README.md
 $ git init project-repo
```

Comment:
This actually creates the "project-repo" working directory, so it must come first.